sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/SQLRunDAO.java

```diff
@@ -502,6 +502,9 @@ public long save(Run run) throws IOException {
               l.setId(sequencerPartitionContainerDAO.save(l));
             }
             SEQ_PART_CONTAINER_WRITER.saveAll(template, run.getId(), run.getSequencerPartitionContainers());
+            for (SequencerPartitionContainer<SequencerPoolPartition> container : run.getSequencerPartitionContainers()) {
+              sequencerPartitionContainerDAO.save(container);
```

Comment:
I idiotically put this in the `saveAll` method where it already existed instead of the `save` method where it is missing. :-/